map/mode.c

```diff
@@ -0,0 +1,81 @@
+#include "map/mode.h"
+#include "common/trinary/tryte_ascii.h"
+#include "mam/mam/mam_channel_t_set.h"
+
+retcode_t map_channel_create(mam_api_t *const api, tryte_t *const channel_id) {
```

Comment:
Why do you use `retcode_t` instead of `status_t`?

Reply:
Because `retcode_t` has more comprehensive error enum. I would like to maintain like this and `status_t` would be used in higher level like TA APIs.